nilearn/signal.py

```diff
@@ -148,6 +149,17 @@ def _detrend(signals, inplace=False, type="linear", n_batches=10):
     return signals
 
 
+def _check_wn(btype, freq, nyq):
+    wn = freq / nyq
```

Comment:
I can do it but it is already implictly casted in line 216: `nyq = sampling_rate * 0.5`